### PR TITLE
adding swagger documentation to project 

### PR DESCRIPTION
**Pull request by MKurshakova**

### CODE REVIEW

**Review by KinWaj — DISMISSED** [review]
;)

File: src/main/java/pl/pjwstk/woloappapi/configs/CorsConfig.java [hunk 1]
```diff
@@ -1,20 +1,19 @@
-package pl.pjwstk.woloappapi;
-
-import org.springframework.context.annotation.Configuration;
-import org.springframework.web.servlet.config.annotation.CorsRegistry;
-import org.springframework.web.servlet.config.annotation.EnableWebMvc;
```
Comment (KinWaj) [inline src/main/java/pl/pjwstk/woloappapi/configs/CorsConfig.java:5]: why is this removed ? :0

**Review by KinWaj — APPROVED** [review]

File: src/main/java/pl/pjwstk/woloappapi/configs/CorsConfig.java (hunk 1 above)
Comment (MKurshakova) [reply src/main/java/pl/pjwstk/woloappapi/configs/CorsConfig.java:5]: We do not need @EnableWebMvc. Spring Boot's auto-configuration for Spring MVC will be applied, and CorsConfig still will be considered.
